Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -2002,31 +1997,17 @@
     CGContextStrokePath();
 
     // Lock the data that is backing the layer.
-    BYTE* buffer = static_cast<BYTE*>(_imgDest->Backing()->LockImageData());
-    THROW_NS_IF_NULL(E_UNEXPECTED, buffer);
-    int stride = _imgDest->Backing()->BytesPerRow();
-    float width = _imgDest->Backing()->Width();
+    _imgDest->Backing()->LockImageData();
```

Comment:
>LockImageData [](start = 25, length = 13)

This lock unlock should actually be in the wicbitmaplock.  Note that the image data might go away after you can ReleaseImageData().